### ISSUE TITLE
update the prototype version of the Density sim to fix the errors in the mystery blocks values

### ISSUE THREAD

**Issue by DianaTavares** [phetsims/density#51]
In #40 some of the mystery values changed, but these modifications are not in the prototype on the website. Can a new prototype version be submitted on the website?  The version on the website has the issue that some of the mystery densities in set 1 don't match with the values in the Density Table. 

#40 is only about the values of the set 1, but reviewing all the density values of the other sets, also set 2, block E2,  in the prototype version on the website, has a wrong value for the mass. It should be mass=44.8 kg and volume=5L, in order of having a density= 8.96 kg/L, corresponding to cooper. 

In summary, all set 1 in the prototype need be updated, and only Block E2 in the set 2.

**Comment by DianaTavares** [comment]
Another user sentt an email about the troubles with the wrong values in the prototype version of the sim (#55) the simulation in the phet-test has all this troubles fixed. Can a new prototype version of the sim be submitted? 
Thanks!

**Comment by DianaTavares** [comment]
this is the same issue that #52  then I am going to close it.
